docs/synapse/userguides/storm_ref_type_specific.rstorm

```diff
@@ -1083,26 +1154,26 @@ Parsing
 
     [ inet:ipv4=1.2.3.4 +#cno.infra.sink.hole=(2017/06/13 09:46+04:00, "?") ]
 
+.. WARNING::
+  
+  Despite storing ``time`` types as epoch millis, Storm does not accept millis format as direct input. For example, it is not possible to input a time value in the format ``1544953072324`` (Storm will attempt to interpret that as the string ``YYYYMMDDhhmmssmmmm`` and return an error).
```

Comment:
we will accept the int ``(1544953072324)`` though.

```cli> storm [inet:ipv4=0 .seen=(1544953072324)]
Executing query at 2021/11/16 16:08:35.918
....
inet:ipv4=0.0.0.0
        .created = 2021/11/16 16:08:36.764
        .seen = ('2018/12/16 09:37:52.324', '2018/12/16 09:37:52.325')
        :type = private```

Reply:
Updated time docs to note support for integerized raw millis format.